controls/radgridview/managing-data/add-new-entries.md

```diff
@@ -54,7 +54,7 @@ The next step in implementing the adding functionality is to attach event handle
 
 ## Modifying New Entries
 
-The __AddingNewDataItem__ event is raised before the new row is added to the RadGridView. A typical use case is when you need to set initial values for the initialized object. This can be done by passing an object to the __GridViewAddingNewEventArgs__'s **NewObject** property.
```

Comment:
@pramodprogress, in this case, the event is raised by RadGridView and not by the developer.
The same applies for line 113.